test/io/serializers/warn-interfaces.good

```diff
@@ -0,0 +1,48 @@
+warn-interfaces.chpl:nnnn: warning: the type 'JustTwoRecord' defines methods that previously had special meaning. These will soon require 'JustTwoRecord' to implement the 'initDeserializable' interface to continue to be treated specially.
+warn-interfaces.chpl:nnnn: note: formerly-special function 'init' defined here
```

Comment:
Along the same lines, I might adjust to: `routine 'init' implicitly belonging to interface defined here`

Where it would be really nice if the word "interface" could be swapped for "initDeserializable" or whatever the actual interface name is.

Reply:
Offline we concluded to save this discussion for another time, since the warnings were pre-existing and apply to other cases as well (e.g. hashable).